api.js

```diff
@@ -6,10 +6,18 @@ var utils = require('./utils');
 if (!('telerivetContext' in global)) {
 
     global.telerivetContext = {
+        // these variables are only defined within Telerivet's script engine in certain contexts (e.g. in response to a message)
+        // they will be undefined in the context of a periodically-running service
+        // https://telerivet.com/api/script#global
+
+        phone: typeof phone === "undefined" ? null : phone,
```

Comment:
@gregstuderoaf here are the mocked global objects + some comments

Reply:
nitpick - we should probably be consistent how we check whether global vars exist - currently we use "'varName' in global"

Feel like that's easier to read than "typeof" - javascript always has like 10 ways of doing this stuff though :-).  

If you agree, maybe change these to 'phone' in global, etc?